app/api/module/Email/src/Service/Email.php

```diff
@@ -3,125 +3,116 @@
 use Psr\Container\ContainerInterface;
-
-/**
- * Class Email
- *
- * @package Olcs\Email\Service
- * @author  Craig Reasbeck <craig.reasbeck@valtech.co.uk>
- */
-class Email implements FactoryInterface
+use Psr\Container\NotFoundExceptionInterface;
+use Symfony\Component\Mailer\Exception\TransportExceptionInterface;
+use Symfony\Component\Mailer\Mailer;
+use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Mailer\Transport;
+use Symfony\Component\Mailer\Transport\Dsn;
+use Symfony\Component\Mailer\Transport\FailoverTransport;
+use Symfony\Component\Mailer\Transport\RoundRobinTransport;
+use Symfony\Component\Mailer\Transport\Smtp\EsmtpTransportFactory;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mime\Email as SymfonyEmail;
+
+class Email
 {
     public const MISSING_FROM_ERROR = 'Email is missing a valid from address';
-    public const MISSING_TO_ERROR = 'Email is missing a valid to address';
-    public const NOT_SENT_ERROR = 'Email not sent: %s';
+    public const MISSING_TO_ERROR   = 'Email is missing a valid to address';
+    public const NOT_SENT_ERROR     = 'Email not sent: %s';
 
-    private $mailTransport;
+    private MailerInterface $mailer;
 
     /**
-     * Get Transport.
-     *
-     * @return TransportInterface
+     * @throws ContainerExceptionInterface
+     * @throws NotFoundExceptionInterface
      */
-    public function getMailTransport()
+    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
     {
-        return $this->mailTransport;
+        $config = $container->get('config')['mail'] ?? null;
+        if (!$config) {
+            throw new \RuntimeException('No mail config found');
+        }
+
+        // Accept either a single transport or an array of transports
+        $items = $config['transports']
+            ?? (isset($config['transport']) ? [ $config['transport'] ] : []);
+
+        if (!is_array($items) || !$items) {
+            throw new \RuntimeException('No mail transports configured');
+        }
+
+        $built = array_map(function (array $item) {
+            $scheme  = $item['scheme'];
+            $host    = $item['host'];
+            $user    = $item['username'];
+            $pass    = $item['password'];
+            $port    = $item['port'];
+
+            $dsn = new Dsn($scheme, $host, $user, $pass, $port);
+
+            // Build transport using Esmtp factory
+            $factory = new Transport([new EsmtpTransportFactory()]);
+            return $factory->fromDsnObject($dsn);
+        }, $items);
+
+
+        // Strategy: first (default), failover, round_robin
+        $strategy = $config['strategy'] ?? 'first';
+        $transport = match ($strategy) {
+            'failover' => new FailoverTransport($built),
+            'round_robin' => new RoundRobinTransport($built),
+            default => $built[0],
+        };
+
+        $this->setMailer(new Mailer($transport));
```

Comment:
This all seems overly complicated. Something like the below is all we need (please check this works first).

Looking further up, for any of this to work at all, it would need to be surrounded by `foreach($items as $item)` - although possibly we just remove multiple transports anyway.

```suggestion
        $dsn = new Dsn(
            $item['scheme'],
            $item['host'],
            $item['username'],
            $item['password'],
            $item['port']
        );

        $factory = new Transport([new EsmtpTransportFactory()]);
        $this->setMailer($factory->fromDsnObject($dsn));
```